src/components/ProgressIndicator/ProgressItem.js

```diff
@@ -5,9 +5,9 @@ import {getBEMClassName} from 'utils';
 
 import CompletionMark from './CompletionMark';
 
-const ProgressItem = ({completed, children}) => {
+const ProgressItem = ({completed, children, modifiers = []}) => {
```

Comment:
Since you added a prop, it is good to also add it to the propTypes on line 20 :slightly_smiling_face: